cockatrice/src/settings/cache_settings.cpp

```diff
@@ -285,6 +285,7 @@ SettingsCache::SettingsCache()
     visualDeckStorageInGame = settings->value("interface/visualdeckstorageingame", true).toBool();
     visualDeckStorageSelectionAnimation =
         settings->value("interface/visualdeckstorageselectionanimation", true).toBool();
+    openInVisualDeckEditor = settings->value("interface/openInVisualDeckEditor", true).toBool();
```

Comment:
I'd suggest using something like `"interface/defaultDeckEditor"` which can be either `"VisualDeckEditor"` or `"DeckEditor"` instead of a boolean toggle, it's both future-proof and self-documenting.

Maybe also having a select box (where you can select either "Deck editor" or "Visual deck editor") in the settings, now that I think about it.